<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSeletorModal</class>
 <widget class="QWidget" name="ColorSeletorModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>604</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="vblSeletorPaletteModal">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wdgColorSeletorBackdrop" native="true">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="vblSeletorPaletteModalBackdrop">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>64</number>
      </property>
      <property name="bottomMargin">
       <number>64</number>
      </property>
      <item>
       <widget class="QWidget" name="wdgColorSeletorCenterModal" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#wdgContainerSeletorPaletteModal {
   background-color: rgb( 255, 255, 255 );
    border: 1px solid grey;
}</string>
        </property>
        <layout class="QHBoxLayout" name="hblGenericCenterModal">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="wdgContainerColorSeletorModal" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>473</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>600</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">* {
background-color: rgb( 255, 255, 255);

}
 
#wdgContainerColorSeletorModal {
    border: 1px solid grey;
}
</string>
           </property>
           <layout class="QVBoxLayout" name="vblContainerSeletorPaletteModal">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>32</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>32</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <widget class="QListWidget" name="lwdCoresPadrao">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>20</horstretch>
                <verstretch>20</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>470</width>
                <height>30</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="autoScrollMargin">
               <number>20</number>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="textElideMode">
               <enum>Qt::ElideMiddle</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerItem</enum>
              </property>
              <property name="movement">
               <enum>QListView::Static</enum>
              </property>
              <property name="flow">
               <enum>QListView::LeftToRight</enum>
              </property>
              <property name="spacing">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGraphicsView" name="gvwVariacaoCor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>470</width>
                <height>290</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
